lib/school_house_web/views/html_helpers.ex

```diff
@@ -35,12 +35,31 @@ defmodule SchoolHouseWeb.HtmlHelpers do
         {"#", "cursor-not-allowed"}
       end
 
-    link(contents,
-      class: "#{class} #{additional_classes}",
-      to: destination
+    additional_classes = if Lessons.coming_soon?(name), do: additional_classes <> " mr-2", else: additional_classes
+
+    content_tag(
+      :div,
+      [
+        link(contents,
+          class: "#{class} #{additional_classes}",
+          to: destination
+        ),
```

Comment:
```suggestion
    class =
      if Lessons.coming_soon?(name) do
        "#{class} #{additional_classes} mr-2"
      else
        "#{class} #{additional_classes}"
      end

    content_tag(
      :div,
      [
        link(contents, class: class, to: destination),
```

This is a more subjective style suggestion - the `if/else` condition on one line is pretty terse and laying it out across multiple lines makes it more obvious what the difference is, in my opinion.

Thoughts on this?